### PR TITLE
Hero banner

### PR DESCRIPTION
**Pull request by IamRokas**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+        <StaticImage src="../images/hero-banner.jpg" alt="Su Ukraina!" />
+        <HeroBanner title="Su Ukraina iki pergalės!" subtitle="Tinklalapis skirtas sutelkti informaciją apie krizę Ukrainoje.">
+          <Constraint customClass="HeroBanner__inner">
+            <CtaCard title="Aukojimas" link="/kaip-galiu-padeti/aukojimas/" iconHandle="donate">
+              <CtaCardItem label="Lietuvoje"/>
+              <CtaCardItem label="Užsienyje"/>
+            </CtaCard>
+            <CtaCard title="Savanoriavimas" link="/kaip-galiu-padeti/savanoryste/" iconHandle="volunteer">
+              <CtaCardItem label="Apgyvendinimas"/>
+              <CtaCardItem label="Pavežėjimass"/>
```
Comment (bmarty92) [inline src/pages/index.js:25]: typo

File: src/components/CtaCard/CtaCardItem.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const CtaCardItem = ({ label }) => {
+  return (
+    <div className="CtaCardItem">
+      {label}
+    </div>
+  );
+};
+
+CtaCardItem.propTypes = {
+  label: PropTypes.node,
```
Comment (bmarty92) [inline src/components/CtaCard/CtaCardItem.js:13]: Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
` label: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.node,
  ]),`

File: src/components/HeroBanner/HeroBanner.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      </div>
+      <div class="HeroBanner__content">
+        {children}
+      </div>
+    </section>
+  );
+};
+
+HeroBanner.propTypes = {
+  bgColor: PropTypes.node,
```
Comment (bmarty92) [inline src/components/HeroBanner/HeroBanner.js:21]: Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
label: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]),

File: src/components/CtaCard/CtaCardItem.js (hunk 2 above)
Comment (grinkus-adapt) [reply src/components/CtaCard/CtaCardItem.js:13]: A string is a node mate :D

File: src/components/Section/Section.js [hunk 4]
```diff
@@ -3,14 +3,15 @@ import PropTypes from "prop-types";
...
-const Section = ({ bgColor, children }) => {
+const Section = ({ customClass, bgColor, children }) => {
   return (
-    <section className={`Section Section--bg-${bgColor}`}>{children}</section>
+    <section className={`Section Section--bg-${bgColor} ${customClass}`}>{children}</section>
   );
 };
 
 Section.propTypes = {
   bgColor: PropTypes.node,
```
Comment (bmarty92) [inline src/components/Section/Section.js:13]: Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
label: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]),

File: src/components/CtaCard/CtaCard.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import "./CtaCard.css";
+
+const CtaCard = ({ children, title, iconHandle, link }) => {
+  return (
+    <div className="CtaCard">
+      <a href={link} title={title}>
```
Comment (grinkus-adapt) [inline src/components/CtaCard/CtaCard.js:9]: Errr... Any way of passing it as a React element? such as `<Link to="/internal/gatsby/link">Text</Link>` or `<a href="http://outside.internet.example.com">Text</a>`? We'd like to keep gatsby prefetching functionality by unsing the internal `Link` if possible

File: src/pages/index.js [hunk 6]
```diff
@@ -1,11 +1,33 @@
 import * as React from "react";
+import Constraint from "../components/Constraint";
```
Comment (bmarty92) [inline src/pages/index.js:2]: group imports

**Review by grinkus-adapt — CHANGES_REQUESTED** [review]

**Comment by bmarty92** [comment]
Information about PropTypes. Very useful ;)
https://reactjs.org/docs/typechecking-with-proptypes.html

File: src/components/CtaCard/CtaCard.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+        <div className="CtaCard__heading">
+          <div className={`CtaCard__icon CtaCard__icon--${iconHandle}`}>
+
```
Comment (bmarty92) [inline src/components/CtaCard/CtaCard.js:12]: Use self closing tag.
 <div className={`CtaCard__icon CtaCard__icon--${iconHandle}`}/>

File: src/components/CtaCard/CtaCard.js [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+          {children}
+          </div>
+      </a>
+    </div>
+  );
+};
+
+CtaCard.propTypes = {
+  children: PropTypes.node,
+  title: PropTypes.node,
```
Comment (bmarty92) [inline src/components/CtaCard/CtaCard.js]: Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
label: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]),

File: src/components/CtaCard/CtaCardItem.js (hunk 2 above)
Comment (bmarty92) [reply src/components/CtaCard/CtaCardItem.js:13]: Got it

**Pull request merged**